build.gradle

```diff
@@ -548,7 +548,7 @@ task gatkDoc(type: Javadoc, dependsOn: classes) {
         options.addStringOption("output-file-extension", phpExtension)
         options.addStringOption("index-file-extension", phpExtension)
     }
-    options.addStringOption("absolute-version", getVersion())
+    options.addStringOption("absolute-version", gitVersion().replaceAll("-SNAPSHOT", "").replaceAll(".dirty", ""))
```

Comment:
If the GATK version is a SNAPSHOT as opposed to a release, shouldn't that be clearly reflected in the docs? I'm not sure I understand the motivation behind this change.